<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(19, 78, 94, 255), stop:1 rgba(113, 178, 128, 255))</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>550</width>
        <height>550</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>550</width>
        <height>550</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>276</width>
        <height>560</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>271</width>
        <height>521</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>276</width>
         <height>548</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QPushButton" name="pushButton_7">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Hero</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 7px;   
background-color: rgba(255, 255, 255, 50)
</string>
          </property>
          <property name="text">
           <string>Cropping photo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_6">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Hero</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 7px;   
background-color: rgba(255, 255, 255, 50)
</string>
          </property>
          <property name="text">
           <string>RESET ALL</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>33</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Hero</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius: 7px;   
background-color: rgba(255, 255, 255, 50)</string>
            </property>
            <property name="text">
             <string>&lt;-    undo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="minimumSize">
             <size>
              <width>109</width>
              <height>33</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Hero</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius: 7px;   
background-color: rgba(255, 255, 255, 50)</string>
            </property>
            <property name="text">
             <string>redo   -&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>259</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>259</width>
              <height>16</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Hero</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 0)</string>
            </property>
            <property name="text">
             <string>size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="verticalSlider">
            <property name="minimumSize">
             <size>
              <width>259</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>259</width>
              <height>22</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 0)</string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>259</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>259</width>
              <height>16</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Hero</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 0)
</string>
            </property>
            <property name="text">
             <string>transparency</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="verticalSlider_2">
            <property name="minimumSize">
             <size>
              <width>259</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>259</width>
              <height>22</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 0)</string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="minimumSize">
             <size>
              <width>153</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>123</width>
              <height>26</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Hero</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius: 7px;   
background-color: rgba(255, 255, 255, 50)</string>
            </property>
            <item>
             <property name="text">
              <string>black and white</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>vertical reflection</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>diagonal reflection</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>viniet</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>glitch</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>create water mark/logo</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gaussian blur</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>blur</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>show edges</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>show better edges</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>99</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Hero</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius: 7px;   
background-color: rgba(255, 255, 255, 50)</string>
            </property>
            <property name="text">
             <string>apply</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>259</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>259</width>
              <height>16</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Hero</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 0)</string>
            </property>
            <property name="text">
             <string>R</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="verticalSlider_3">
            <property name="minimumSize">
             <size>
              <width>259</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>259</width>
              <height>22</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 0)</string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>259</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>259</width>
              <height>16</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Hero</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 0)</string>
            </property>
            <property name="text">
             <string>G</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="verticalSlider_4">
            <property name="minimumSize">
             <size>
              <width>259</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>259</width>
              <height>22</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 0)</string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>259</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>259</width>
              <height>16</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Hero</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 0)</string>
            </property>
            <property name="text">
             <string>B</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="verticalSlider_5">
            <property name="minimumSize">
             <size>
              <width>259</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>259</width>
              <height>22</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 0)</string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="minimumSize">
             <size>
              <width>123</width>
              <height>39</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>123</width>
              <height>39</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Hero</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 0)</string>
            </property>
            <property name="text">
             <string>processing by cores</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="minimumSize">
             <size>
              <width>122</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>122</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Hero</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius: 7px;   
background-color: rgba(255, 255, 255, 50)</string>
            </property>
            <property name="text">
             <string>apply</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="minimumSize">
             <size>
              <width>175</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>175</width>
              <height>20</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>78</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="minimumSize">
             <size>
              <width>83</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Hero</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius: 7px;   
background-color: rgba(255, 255, 255, 50)</string>
            </property>
            <property name="text">
             <string>Save image</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="test.qrc"/>
 </resources>
 <connections/>
</ui>
